pkgs/top-level/all-packages.nix

```diff
@@ -3542,6 +3542,8 @@ in
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
+  pax = callPackage ../tools/archivers/pax {};
```

Comment:
```suggestion
  pax = callPackage ../tools/archivers/pax { };
```

style nitpick

Reply:
We have both variants in all-packages.nix, but I don't have a strong opinion about the number of spaces between empty braces.